### DISCUSSION TITLE
Vertical tab - is this the right usage?

### DISCUSSION THREAD

**Discussion by charleneyjh1** [twilio-labs/paste#1294] [Q&A] [answered]
Hi Dsys!

Is this the right usage of the vertical tab component? Any feedback or suggestions?

<img width="680" alt="Screen Shot 2021-03-16 at 3 45 01 PM" src="https://user-images.githubusercontent.com/56697620/111384284-f0d5cc80-866e-11eb-8f45-a482465822ac.png">

**Comment by TheSisb** [comment, marked as answer]
Hi @charleneyjh1,

What kind of feedback are you looking for in terms of vertical tab usage on this page? Strictly looking at the tabs in isolation, it appears to be used correctly. If you're asking about the Tabs in relation to the rest of the page, then I don't think the design system team is close enough to your product to really be able to provide valuable insight with the information provided.  Was there any specific feedback you wanted to run by us that you received during design crit?

**Comment by charleneyjh1** [comment]
Thanks Shadi,  I was advised to used vertical tabs component in office hour. However I haven't seen it used within a card before. Just wanna to check with Dsys is it the right usage. You've answered my question, thank you.
